ci/build/build-standalone-release.sh

```diff
@@ -28,8 +28,31 @@ main() {
   ln -s "./lib/node" "$RELEASE_PATH/node"
 
   pushd "$RELEASE_PATH"
-  yarn --production --frozen-lockfile
+  yarn --frozen-lockfile
+  create_shrinkwraps
   popd
 }
 
+create_shrinkwraps() {
+  # yarn.lock or package-lock.json files (used to ensure deterministic versions of dependencies) are
+  # not packaged when publishing to the NPM registry.
+  # To ensure deterministic dependency versions (even when code-server is installed with NPM), we create
+  # an npm-shrinkwrap.json file from the currently installed node_modules. This ensures the versions used
+  # from development (that the yarn.lock guarantees) are also the ones installed by end-users.
+
+  # We first generate the shrinkwrap file for code-server itself - from being in $RELEASE_PATH
+  npm shrinkwrap
+
+  # Then the shrinkwrap files for the bundled VSCode
+  # We don't need to remove the devDependencies for these because we control how it's installed - and
```

Comment:
> We don't need to remove the devDependencies for these

That makes sense, does that mean we should be removing them for the root dependencies?  It looks like we are including dev deps right now in the root shrinkwrap (I might be missing something!).

Reply:
So the expectation is that when installing something with `--global` infers only keeping the prod dependencies. So rather than manipulating the shrinkwrap to remove them, we should be clear to leave them there and let `npm` do its thing.

That being said, there seems to be another nasty bug with that... https://github.com/npm/cli/issues/4323

But I'm not sure that removing them from the shrinkwrap won't simply cause NPM to try to download them anyway... Let me play around!

Reply:
Ohhhh that makes sense!  Definitely agreed that we should just let `npm` do its thing.